core/modules/qdisp/Executive.cc

```diff
@@ -168,6 +145,8 @@ void Executive::add(int refNum, Executive::Spec const& s) {
         return;
     }
     ExecStatus& status = _insertNewStatus(refNum, s.resource);
+    _empty.set(false);
+    LOGF_DEBUG("Flag set to false %1%" % _empty.get());
```

Comment:
log4cxx official documentation show a good way to fine tune the logger. An example is here: https://jira.lsstcorp.org/browse/DM-2373.
I'm not sure this should be done in the current ticket.